### PR TITLE
schematic: Fix dereference before NULL check (coverity scan issue 209544).

### PR DESCRIPTION
**Pull request by vzh**

### CODE REVIEW

File: schematic/src/o_move.c [hunk 1]
```diff
@@ -539,9 +539,6 @@ void o_move_check_endpoint(GschemToplevel *w_current, OBJECT * object)
   PAGE *page = gschem_page_view_get_page (page_view);
   g_return_if_fail (page != NULL);
 
-  if (!object)
-  return;
```
Comment (graahnul-grom) [inline schematic/src/o_move.c:543]: Wouldn't it be better to move this check to the beginning of the function?

Comment (vzh) [reply schematic/src/o_move.c:543]: Yes, you're right.  It's better to amend the check.

Comment (vzh) [reply schematic/src/o_move.c:543]: Please check the recent version.

Comment (graahnul-grom) [reply schematic/src/o_move.c:543]: OK

**Pull request merged**